addons/stock/wizard/stock_immediate_transfer_views.xml

```diff
@@ -6,7 +6,7 @@
         <field name="arch" type="xml">
             <form string="Immediate transfer?">
                 <group>
-                    <p>You have not recorded <i>done</i> quantities yet, by clicking on <i>apply</i> Odoo will process all the quantities.</p>
+                    <p>You have not recorded <i>done</i> quantities yet, by clicking on <i>apply</i> UDES will process all the quantities.</p>
```

Comment:
Suggestion: Make this a bit more "neutral", I'm sure about a direct reference to UDES in Odoo (even if it is our branch of Odoo).

Maybe:

`You have not recorded done quantities yet, by clicking on apply all outstanding quantities will be processed.`?

Reply:
Yeah I agree, this should be fixed.